### PR TITLE
install Bosun binary from Github releases page

### PR DESCRIPTION
**Pull request by timurb**

### CODE REVIEW

File: recipes/golang.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Use golang recipe to install bosun
+default['go']['packages']           = ['bosun.org/cmd/bosun']
+default['bosun']['install_path']    = "#{node['go']['gobin']}/bosun"
```
Comment (ptqa) [inline recipes/golang.rb]: is it really allowed to set attributes like that in recipe?

Comment (timurb) [inline recipes/golang.rb]: Right catch, thanks!. It should be as `node.default`. I'll update the pull request shortly.

File: attributes/default.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # This cookbook support only runit
 default['bosun']['init_style']      = 'runit'
+default['bosun']['install_style']   = 'binary' # binary or golang
```
Comment (ptqa) [inline attributes/default.rb:3]: Maybe it would be better to use attribute version (latest = golang, else binary) for that?

Comment (timurb) [inline attributes/default.rb:3]: I think installation of Golang and compilation from source should be made more explicit than that.

Comment (ptqa) [inline attributes/default.rb:3]: Ok, let it be.

**Comment by ptqa** [comment]
Thanks for contributing!

**Pull request merged**